storage/ceph.py

```diff
@@ -1,14 +1,22 @@
 import pyarrow.parquet as pq
-import s3fs
+import s3fs, datetime, os, s3, local
 
 (ceph_key, ceph_secret, ceph_endpoint) = (os.getenv('CEPH_KEY'), os.getenv('CEPH_SECRET'), os.getenv('CEPH_ENDPOINT'))
 client_kwargs = {'endpoint_url': ceph_endpoint}
 
 
 def get_dataset(key):
-    s3 = s3fs.S3FileSystem(secret=ceph_secret, key=ceph_key, client_kwargs=client_kwargs)
-    return pq.ParquetDataset(key, filesystem=s3).read_pandas().to_pandas()
+    filesystem = s3fs.S3FileSystem(secret=ceph_secret, key=ceph_key, client_kwargs=client_kwargs)
+    return pq.ParquetDataset(key, filesystem=filesystem).read_pandas().to_pandas()
 
 
-def write(data, date):
-    raise NotImplementedError
+def write(data, use_local=True):
+	date = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+	if use_local:
```

Comment:
dont use the local switch. Just write the data. And the `read` method below has to be implemented as well.

Reply:
So here I should just use `local.write()`?

Reply:
nope. this `ceph` layer will only operate on `ceph` storage. never on `local` storage. Only https://github.com/RedHatInsights/aicoe-insights-clustering/blob/master/storage/local.py will do this